droid-help/src/main/resources/Web pages/Command line control.html

```diff
@@ -1355,7 +1400,16 @@ <h3> <a id="OptionsTable" name="OptionsTable">Command options</a></h3>
         <td style="border:1px solid #000" bgcolor="#000000"><font color="#ffffff" face="Monospaced, Courier New, Courier, mono"/>
             droid -s 97
             <p>
-            droid -s 20200121
+            droid
+        </td>
+    </tr>
+    <tr>
+        <td>-json</td>
+        <td>--json-output</td>
+        <td>{none}</td>
+        <td>Sets the output to json. This will work for any command which outputs to console or to a file</td>
```

Comment:
"JSON" is lower cased here but elsewhere it is upper cased

Reply:
I've changed it to upper case.